com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -592,6 +592,25 @@ private void ProcessNetworkBehaviour(TypeDefinition typeDefinition, string[] ass
                 instructions.Reverse();
                 instructions.ForEach(instruction => processor.Body.Instructions.Insert(0, instruction));
             }
+
+            // __getBehaviourName processor
+            {
```

Comment:
Why is this scoped (i.e. in brackets)? Is this an ILPP thing or something else?

Reply:
I think its just an documentation/organizational thing, since the ILPP code can be very hard to follow

Reply:
yeah, a block with comment on top is a cool way to document what's going on in that region.

Reply:
in addition to andrew and fatih's comments, the brackets are to scope the local variables since they share a name with the other scopes. reasonably these could be functions to avoid this but i didn't want to add complexity to an already huge class